mne/io/meas_info.py

```diff
@@ -518,8 +472,9 @@ def _get_channel_positions(self, picks=None):
         -----
         .. versionadded:: 0.9.0
         """
-        picks = _picks_to_idx(self.info, picks)
-        chs = self.info["chs"]
+        info = self if isinstance(self, Info) else self.info
+        picks = _picks_to_idx(info, picks)
+        chs = info["chs"]
```

Comment:
it is exactly the small changes like this that are easy to see when the copy-paste is done in a separate commit, and very hard to see otherwise.  (this looks fine, BTW)